src/global.ts

```diff
@@ -80,7 +81,7 @@ interface GamePF2e
         UserPF2e
     > {
     pf2e: {
-        actions: Record<string, Function>;
+        actions: Record<string, Function> & Map<string, Action>;
```

Comment:
How is this both a `Map` and a `Record`?

Reply:
It has the old action functions directly on the object (the `Record` part) _and_ it’s an instance of a Foundry `Collection`. I couldn’t get the types right for the Collection part, though.

Reply:
I managed to change the type to `Record<string, Function> & Collection<Action>`. Not sure why it didn't work when I tried the first time 🤷

The long-term goal is for it to be only a Collection and all the old action functions to go away. They are currently used by the action macros and even a few places in the system itself, so we'd need some sort of deprecation period and/or migration.